components/Main/ContactUs/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react'
```

Comment:
As mentioned above this does seem similar to the `NewsletterSignUp` component...